### ISSUE TITLE
violation on schema:accessibilityFeature resizeText

### ISSUE THREAD

**Issue by gregoriopellegrino** [daisy/ace#280]
- Ace by DAISY 1.0.0-rc.1 (GUI)
- MacOs 10.14.5

I get the violation `metadata-accessibilityfeature-invalid` for the metadata:
`<meta property="schema:accessibilityFeature">resizeText</meta>`

But `resizeText` is listed in the KB: http://kb.daisy.org/publishing/docs/metadata/schema-org.html#accessibilityFeature

**Comment by rdeltour** [comment]
This issue came up recently in [another comment](https://github.com/daisy/ace/issues/242#issuecomment-524768585). TL;DR: the value `resizeText` is not listed in the authoritative [W3C WebSchemas/Accessibility wiki page](https://www.w3.org/wiki/WebSchemas/Accessibility#Accessibility_terms_.28Version_2.0.29).

I’ll file an issue to update the KB.

**Comment by rdeltour** [comment]
I’m closing the issue as `invalid`, but please feel free to keep on commenting here or reopen if you think Ace should do differently!

**Comment by rdeltour** [comment]
KB issue filed as daisy/kb#21
